paddlets/ensemble/stacking_ensemble_forecaster.py

```diff
@@ -40,12 +40,12 @@ def __init__(self,
                  out_chunk_len: int,
                  skip_chunk_len: int,
                  estimators: List[Tuple[object, dict]],
-                 final_learner: Callable = None,
-                 use_backtest: bool = True,
-                 resampling_strategy: str = 'cv',
-                 split_ratio: Union[str, float] = 0.1,
-                 k_fold: Union[str, int] = 3,
-                 verbose: bool = False
+                 final_learner: Optional[Callable] = None,
+                 use_backtest: Optional[bool] = True,
+                 resampling_strategy: Optional[str] = 'cv',
+                 split_ratio: Optional[Union[str, float]] = 0.1,
+                 k_fold: Optional[Union[str, int]] = 3,
+                 verbose: Optional[bool] = False
```

Comment:
Line44 - Line48  use_backtest, resampling_strategy, split_ratio, split_ratio, k_fold, verbose should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional